app/workers/delinquent_finder.rb

```diff
@@ -0,0 +1,16 @@
+class DelinquentFinder
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  recurrence { daily }
+
+  def perform
+    late_unmarked_reservations = Reservation.joins(gift_request: {list: :event}).includes(donor: :user)
+                                   .where("events.end_date <= ?", Date.today)
+                                   .where(delinquent: false)
```

Comment:
Should this also have a clause that only checks for `:reserved` or `:shipped` reservations (i.e. "received" reservations aren't marked delinquent?) or is that handled somewhere else?